content/en-US/api/SerialPort.md

```diff
@@ -0,0 +1,124 @@
+---
+recipe: api-interface
+title: 'SerialPort'
+mdn_url: /en-US/docs/Web/API/SerialPort
+specifications: https://wicg.github.io/serial/#serialport-interface
+browser_compatibility: api.SerialPort
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+// Complete the sentence below with a basic description of the API.
+// You may add more sentences if needed.
+
+The `SerialPort` interface of the Web Serial API provides access to a serial
+port on the host device. Instances of this interface may be obtained by calling
+`navigator.serial.requestPort()` to request permission to access a port from the
+user. Ports that an origin has already been granted permission to access are
+available from `navigator.serial.getPorts()` or by listening for `"connect"`
+events fired at `navigator.serial`.
+
+This interface requires a secure context.
+
+## Constructor
+
+None.
+
+## Properties
+
+**`SerialPort.readable`**
+
+Returns a `ReadableStream` for receiving data from the device connected to the
+port. Chunks read from this stream will be instances of `Uint8Array`. This
+property will be non-null as long as the port is open and has not encountered a
+fatal error.
+
+**`SerialPort.writable`**
+
+Returns a `WritableStream` for sending data to the device connected to the port.
+Chunks written to this stream must be instances of `BufferSource` (e.g. an
+`ArrayBuffer` or `ArrayBufferView` such as `Uint8Array`). This property will be
+non-null as long as the port is open and has not encountered a fatal error.
+
+## Events
+
+**`SerialPort.onconnect`**
+
+Called when the port has been connected to the device. This event is only fired
+for ports associated with removable devices such as those connected via USB.
+This event bubbles to the instance of `navigator.serial` which returned this
```

Comment:
`navigator.serial` is the target, right? Why not use that term.

Reply:
The `SerialPort` is the target. The event bubbles from the `SerialPort` to the `navigator.serial` that returned it. This is a change from the WebUSB API where we defined a new event interface which is fired at `navigator.usb` only. In the Web Serial API we fire a generic `Event` at the `SerialPort` and define the bubbling behavior.